app.js

```diff
@@ -3,6 +3,7 @@ const express = require('express');
 const {getTopics} = require('./controllers/topic-controller');
 const {getApiData} = require('./controllers/core-api-controller');
 const {getArticles, getArticleById} = require('./controllers/article-controller');
+const {getCommentsByArticle} = require('./controllers/comment-controller');
 
 const app = express();
 app.use(express.json());
```

Comment:
This really shouldn't be here yet - you haven't written anything involving a body so far, so there is no need for this :) We should really work towards the specification of the tickets we are given, not act on assumptions